pennylane/plugins/default_gaussian.py

```diff
@@ -765,13 +765,6 @@ def apply(self, operation, wires, par):
             self._state = self._operation_map[operation](*par, hbar=self.hbar)
             return # we are done here
 
-        if operation == 'Interferometer':
-            if par[0].shape[0] != len(wires):
-                raise ValueError("Interferomer unitary matrix applied to the incorrect "
```

Comment:
Isn't this still a useful error to raise?

Reply:
It's been superseded by the more general one on line 812, that works for any operation (and doesn't need to single out the interferometer)